expandybird/expander/expander.go

```diff
@@ -46,6 +46,30 @@ func NewExpander(binary string) Expander {
 	return &expander{binary}
 }
 
+// NewTemplateFromType creates and returns a new template whose content
+// is a YAML marshaled resource assembled from the supplied arguments.
+func NewTemplateFromType(name, typeName string, properties map[string]interface{}) (*common.Template, error) {
```

Comment:
Yeah, I should've made it clear, I don't think they belong in this package at all :) Could you open an issue to move them out of here and I can clean that up?